src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -141,13 +142,32 @@ void RegisterContainerAdapter(IContainer containerToAdapt)
                 }
 
                 var username = GetInstallationUserName();
+                await CreateQueuesIfNecessary(receiveInfrastructure, username).ConfigureAwait(false);
```

Comment:
Good for me too. Isn't the consequence though thah we check for send-only multiple times? (Obviously we did that before so not a biggy)